packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -1360,3 +1361,103 @@ class KeepAlive extends ParentDataWidget<KeepAliveParentDataMixin> {
     properties.add(DiagnosticsProperty<bool>('keepAlive', keepAlive));
   }
 }
+
+// Return a Widget for the given Exception
+Widget _createErrorWidget(Object exception, StackTrace stackTrace) {
+  final FlutterErrorDetails details = FlutterErrorDetails(
+    exception: exception,
+    stack: stackTrace,
+    library: 'widgets library',
+    context: ErrorDescription('building'),
+  );
+  FlutterError.reportError(details);
+  return ErrorWidget.builder(details);
+}
+
+/// Set a cross axis extent constraint to the child Sliver. By default, slivers will try to
+/// fill up the available cross axis extent, but this parent data widget will override that
+/// behavior.
+class SliverConstrainedCrossAxis extends ParentDataWidget<SliverFlexParentData> {
+  /// Constructor.
```

Comment:
See some of the other slivers in this file for an example of documenting this.

Reply:
I added a description for the constructor!